apps/frontend/src/app/2_molecules/FixedInterestPoolRowTitle/FixedInterestPoolRowTitle.tsx

```diff
@@ -33,7 +33,7 @@ export const FixedInterestPoolRowTitle: FC<FixedInterestPoolRowTitleProps> = ({
       <span className="text-gray-30 ml-1 font-medium">
         {isBorrow
```

Comment:
@john-light - this condition was there to distinguish between borrow/lend and to display `APR`/`APY` accordingly. Was that incorrect and should we use `APR` now everywhere?

Reply:
@tiltom  you should use APR in the places where it is specified to use APR.